CIP-0XXX/README.md

```diff
@@ -0,0 +1,540 @@
+---
+CIP: ?
+Title: Cardano Decentralized Message Queue
+Category: ?
```

Comment:
> if all the network behaviour described herein is in the consensus layer

Categorizing this CIP will be interesting and we will likely encounter this scoping discussion several times throughout review of this CIP:

We saw three protocols with a need for this CIP: Mithril, Praos and Leios. Only the latter two are scoped to be used by the cardano node itself for consensus. In any case, the goal of this CIP is not to define the consensus protocols, but rather to standardize the network protocol used for message submission.

Reply:
If anything, it seems to me the right category for this CIP should be `Network`

Reply:
@abailly the CIP editors would be really happy to see a `Network` category: especially if you & your team would also identify any previous CIPs & drafts which would be eligible for it.

`Consensus` is proceeding as a CIP category due to the work that @dnadales mentioned here (as far as I know, the terms "enroll" and "enlist" are being used interchangeably):
- https://github.com/cardano-foundation/CIPs/pull/872#discussion_r1706965548
- https://github.com/IntersectMBO/ouroboros-consensus/issues/1205

... of documenting it with a CIP and fulfilling other expectations of [enlistment](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0001/README.md#project-enlisting).  We would love to hear that something similar could happen / is happening for a new `Network` category.

Reply:
We shouldn't be held by review of all the previous CIPs, but the core network protocols haven't had any CIP so far, at least (co-)authored by the networking team.  It could be that there was a CIP to add the `LocalTxMonitor` mini-protocol (done in this PR IntersectMBO/ouroboros-network#3404 by @KtorZ) - that would be a good candidate if there was a CIP for that.

Reply:
@coot Of course, what happened happened. I am only suggesting that at least this CIP, and possibly others to come in the future, would be better categorised as `Network` and therefore it would be great if there was such a category, with some rules and guidelines.

Reply:
> We shouldn't be held by review of all the previous CIPs

@coot I wasn't suggesting that you be obligated to review them: only that CIP Editors would want to identify any "Network" CIPs you could point out, and likely assign a `Network` category to those retroactively if & when it becomes an enlisted category.

Reply:
If you identify such list, I can review it if you'd need some help.

Reply:
Whilst we are [Rethinking Project Enlisting](https://github.com/cardano-foundation/CIPs/issues/898)
Im happy to move and assign `Networking` for now, with the understanding that we are working on how we navigate project enlistment in a post-chang world.

Reply:
@coot @jpraynaud @abailly would you prefer `Network` or `Networking`?

Reply:
`Network` seems fine to me :+1:

Reply:
thanks @coot @jpraynaud @abailly - I've amended it above (from `Consensus`) and we'll collect similar proposals under the `Network` category term (unless `Consensus` is better applicable for those)... though like we said at yesterday's meeting there aren't likely to be many.  cc @Ryun1

Reply:
Done in https://github.com/cardano-scaling/CIPs/commit/e44e9ac1315264db6f8d9f9f2bedb9d40bcae38c :heavy_check_mark: